samples/emoji-search/android-composeui/src/main/kotlin/com/example/redwood/emojisearch/android/composeui/EmojiSearchApplication.kt

```diff
@@ -1,32 +0,0 @@
-/*
- * Copyright (C) 2024 Square, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.example.redwood.emojisearch.android.composeui
-
-import android.app.Application
-import coil3.ImageLoader
-import coil3.PlatformContext
-import coil3.SingletonImageLoader
-import coil3.fetch.NetworkFetcher
-
-class EmojiSearchApplication : Application(), SingletonImageLoader.Factory {
-  override fun newImageLoader(context: PlatformContext): ImageLoader {
-    return ImageLoader.Builder(context)
-      .components {
-        add(NetworkFetcher.Factory())
```

Comment:
As of alpha02 Coil imports this by default if it's on the classpath. It uses the same mechanism that Ktor uses to detect engines. Can be opted out of using `ImageLoader.Builder.serviceLocatorEnabled(false)`.

Reply:
Nice. In a real app I would load it manually still, but this is fine for the samples.

Reply:
Yep, I was unsure about adding this feature, but it's there if folks want it. Adding components manually always takes precedence over automatically loaded components.